common/djangoapps/student/views.py

```diff
@@ -1804,6 +1805,28 @@ def create_account_with_params(request, params):
     return new_user
 
 
+def _enroll_user_in_pending_courses(user_id):
+    """
+    Enroll student in any pending courses he/she may have.
+    """
+    student = User.objects.filter(id=user_id)
```

Comment:
in L1778, we have access to the `User` object. Why not just pass it directly into `_enroll_user_in_pending_courses` so we don't have to do this extra query?